tests-performance/src/Makefile

```diff
@@ -14,19 +14,28 @@ CFLAGS=-std=c++17 \
   -fdiagnostics-show-option \
   -g
 
-
-INCLUDE=-I../../include -I../../3rdparty/simlib/include -I../../3rdparty/re2/include -I ../../3rdparty -I ../../3rdparty/cudd/cudd
-
-LIBS_ADD=-L../../build/src -L../../build/3rdparty/re2 -L../../build/3rdparty/simlib -L../../build/3rdparty/cudd/lib
-
+INCLUDE_DIR=../../include
+THIRD_PARTY_DIR=../../3rdparty
+BUILD_DIR := $(shell \
+	if [ -d "../../cmake-build-release" ]; then \
+		echo "../../cmake-build-release"; \
+	elif [ -d "../../cmake-build-debug" ]; then \
+		echo "../../cmake-build-debug"; \
+	else \
+		echo "../../build"; \
```

Comment:
If we are starting to use the built-in CMake variables, we should gradually remove all uses of relative paths in Makefiles. I would prefer if we do not introduce new relative paths sch as these with `../../` at least. We can use the root dir CMAKE variable instead.